### ISSUE TITLE
Can't take away items from Item Frames in creative mode (client bug?)

### ISSUE THREAD

**Issue by MrPowerGamerBR** [Nukkit/Nukkit#1425]
### Issue description
Can't take away items from Item Frames, after debugging, it seems the ITEM_FRAME_DROP_ITEM_PACKET isn't triggered in creative mode. (Client bug?)
### Steps to reproduce the issue

1. Place an item frame
2. Place a item inside the item frame
3. Change to creative mode
4. Try to take out the item, the item is taken client side but not server side

### OS and versions
* Nukkit Version: This server is running Nukkit 1.0dev ???(Apple)?(Pie)? implementing API version 1.0.0 for Minecraft: PE v1.0.0 (protocol version 100)
 
* Java Version: Java 8

* Host OS: Windows 10

* Host Configuration: 8GB RAM, 1TB HDD, 6 core CPU from AMD

* Client Edition: Windows 10

* Client Version: 1.0.2

### Crashdump, backtrace or other files
No errors

**Comment by MrPowerGamerBR** [comment]
Client issue https://github.com/pmmp/PocketMine-MP/issues/339

**Comment by dktapps** [comment]
No, this is not a client issue, but an incorrect implementation on the server-side.

**Comment by dktapps** [comment]
Correction: this is a client bug, it also occurs in vanilla multiplayer. The client also does not send left-click actions when removing items from frames, so there is no way whatsoever to detect this in creative.

**Comment by MrPowerGamerBR** [comment]
@dktapps TL;DR: #blameshoghi

Anyway, thanks for confirming that this is actually a client side issue 👍

**Comment by dktapps** [comment]
https://bugs.mojang.com/browse/MCPE-20070
